src/rulesets/Base/arraymath.jl

```diff
@@ -434,6 +434,16 @@ function rrule(::typeof(-), x::AbstractArray)
     return -x, negation_pullback
 end
 
+#####
+##### Subtraction
+#####
+
+frule((_, Δx, Δy), ::typeof(-), x::AbstractArray, y::AbstractArray) = x-y, Δx-Δy
+
+function rrule(::typeof(-), x::AbstractArray, y::AbstractArray)
+    subtract_pullback(dy) = (NoTangent(), dy, -dy)
+    return x-y, subtract_pullback
```

Comment:
the autoformatter is probably complaining about spaces around binary operators

```suggestion
frule((_, Δx, Δy), ::typeof(-), x::AbstractArray, y::AbstractArray) = (x - y), (Δx - Δy)

function rrule(::typeof(-), x::AbstractArray, y::AbstractArray)
    subtract_pullback(dy) = (NoTangent(), dy, -dy)
    return x - y, subtract_pullback
```

Reply:
yep, fixed already!:)